scripts/feature_tests/upgrade/combined_tests/1cp_1w_bootDiskImageANDK8sControllers_clusterLevel_upgrade.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+
+set -x
+
+IRONIC_IMAGE_TAG=master
+
+METAL3_DEV_ENV_DIR="$(dirname "$(readlink -f "${0}")")/../../../../"
+# shellcheck disable=SC1091
+# shellcheck disable=SC1090
+source "${METAL3_DEV_ENV_DIR}/scripts/feature_tests/upgrade/common.sh"
```

Comment:
I would rather rename `common.sh `to something that doesn't interfere with `lib/common.sh`

Reply:
done in PR 357. needs to be handled when this PR is merged.